en/full-node.md

```diff
@@ -190,7 +190,7 @@ have an easy-to-use node.
 * Desktop or laptop hardware running recent versions of Windows, Mac OS
   X, or Linux.
 
-* {{site.text.bitcoin_datadir_gb}} gigabytes of free disk space,
+* {{site.text.bitcoin_datadir_gb_pruned}} gigabytes of free disk space,
```

Comment:
Note: Bitcoin Core GUI currently only allows pruning down to 2 GB (+5 GB overhead = 7 GB). So the 5 GB here only holds if the user chooses Knots, or knows how to configure and use bitcoind.

Reply:
Altered to 7GB in f509116.